### PR TITLE
PR v0.0.4 WrenAI kustomization manifests for k8s by Damien Berezenko https://medium.com/@qdrddr

### PR DESCRIPTION
**Pull request by qdrddr**
PR v0.0.4 WrenAI kustomization manifests for k8s by Damien Berezenko https://medium.com/@qdrddr

Fixing some bugs & versions, updating versions. 

Can you please include this into the 0.4.0-rc.1 and the upcoming 0.4.0? @wwwy3y3

### CODE REVIEW

File: deployment/kustomizations/patches/_gitignore
Comment (wwwy3y3) [inline deployment/kustomizations/patches/_gitignore:1]: If we don't need gitignore, intead of renaming this file, maybe we could just remove this ?

**Review by wwwy3y3 — COMMENTED** [review]
Could we explain a bit what the files in `deployment/kustomizations/patches` for ?

Other LGTM.

**Comment by qdrddr** [comment]
> Could we explain a bit what the files in `deployment/kustomizations/patches` for ?
> 
> Other LGTM.

These are the files that are not used but can help a beginner to adjust the kustomization to their needs in their k8s cluster.

Kustomization tool allows to build “layers” of one kustomization that using another kustomization that uses another customization. And so on. This is helpful in case people lavarege GitOps tools such as ArgoCD or FluxCD tools for k8s deployment.

So people in production tend to use the official deployments unmodified directly from the for repo. And then on top of that create a layer that simply customizes only some parts of the official deployment.

So additional patches intended to be modified by the user while others will be used directly from the git repo. It’s useful for DevOps people. Hope that helps

File: deployment/kustomizations/patches/_gitignore
Comment (qdrddr) [reply deployment/kustomizations/patches/_gitignore:1]: Sure, I can remove it.

**Comment by wwwy3y3** [comment]
@qdrddr thanks for the explanation. Could you put that in readme as well? Thanks

**Comment by qdrddr** [comment]
> explanation. Could you put that in readme as well? Thank

Done

File: deployment/kustomizations/patches/_gitignore
Comment (qdrddr) [reply deployment/kustomizations/patches/_gitignore:1]: done

**Comment by qdrddr** [comment]
All done @wwwy3y3

**Comment by qdrddr** [comment]
Can you please include this into the 0.4.0-rc.1 and the upcoming 0.4.0? @wwwy3y3

**Review by wwwy3y3 — APPROVED** [review]
LGTM

**Pull request merged**